appveyor.yml

```diff
@@ -1,31 +1,25 @@
-version: 1.0.{build}
 os: Visual Studio 2015
-before_build:
-- git submodule update --init
-- nuget restore Avalonia.sln
-
+platform:
+- Any CPU
+configuration:
+- Release
 environment:
-  myget_key:
+  NUGET_API_KEY:
+    secure: ADD_SECURE_NUGET_API_KEY_HERE
+  NUGET_API_URL: https://www.nuget.org/api/v2/package
+  MYGET_API_KEY:
     secure: XOgD5bJUKNOS2kDDgb+affS4pDcslxALh+xvvnr1Koy0PjXlhILsBdNhxRe0KcNm
```

Comment:
Not bad.

https://www.appveyor.com/docs/build-configuration/#secure-variables

"AppVeyor generates a unique encryption key for every account. To encrypt variable values go to Account -> Encrypt data tool."

The api keys are encrypted and tied to appveyor account.

We can remove those and set them through appveyor setting panel.